chat/src/components/chat/multimodal-input.tsx

```diff
@@ -92,19 +92,37 @@ function PureMultimodalInput({
 
   const submitForm = useCallback(async () => {
     if (newSession) {
-      await fetchClientWithThrow.POST("/api/sessions", {
-        body: {
-          id: sessionId,
-          title: input.split(" ").reduce((acc, word) => {
-            if (acc.length + word.length + 1 <= 15) {
-              return acc + (acc ? " " : "") + word;
-            }
-            return acc;
-          }, ""),
-          agent_id: agentId,
-        },
-      });
-      queyrClient.invalidateQueries({ queryKey: ["get", "/api/sessions", {}] });
+      try {
+        await fetchClientWithThrow.POST("/api/sessions", {
+          body: {
+            id: sessionId,
+            title: input.split(" ").reduce((acc, word) => {
+              if (acc.length + word.length + 1 <= 15) {
+                return acc + (acc ? " " : "") + word;
+              }
+              return acc;
+            }, ""),
+            agent_id: agentId,
+          },
+        });
+        queyrClient.invalidateQueries({
+          queryKey: ["get", "/api/sessions", {}],
+        });
+      } catch (error: any) {
```

Comment:
@ErezSha this code is not needed, better to parse the Error as JSON in https://github.com/flux0-ai/flux0/blob/develop/chat/src/lib/api/api.ts#L10